apps/frontend/src/components/searchBar.tsx

```diff
@@ -0,0 +1,252 @@
+import {
+  Grid,
+  Button,
+  TextField,
+  Menu,
+  MenuItem,
+  useMediaQuery,
+} from "@mui/material";
+import martletPlaceLogo from "../images/martletplace-logo.png";
+import message from "../images/message.png";
+import filter from "../images/filter.png";
+import { useStyles } from "../styles/pageStyles";
+import { useState, ChangeEvent, useEffect, useCallback } from "react";
+import Filters from "./filters";
+import { useNavigate } from "react-router-dom";
+import * as React from "react";
+
+interface SearchObject {
+  query: string;
+  minPrice: number;
+  maxPrice: number;
+  status: string;
+  searchType: string;
+  latitude: number;
+  longitude: number;
+  sort: string;
+  page: number;
+  limit: number;
+}
+
+interface SearchBarProps {
+  onSearch: () => void;
+  sortBy: string;
+}
+
+const SearchBar: React.FC<SearchBarProps> = ({ onSearch, sortBy }) => {
+  const classes = useStyles();
+  const navigate = useNavigate();
+
+  const [showFilters, setShowFilters] = useState(false);
+  const [searchInput, setSearchInput] = useState("");
+  const [filters, setFilters] = useState<SearchObject>({
+    query: "",
+    minPrice: 0,
+    maxPrice: 100000,
```

Comment:
these max/min prices should either be null initially, or these values should be reflected in the filters component

Reply:
changed the min/max prices to initially be null